test/integration/conanfile/test_deprecated.py

```diff
@@ -63,3 +65,18 @@ def configure(self):
 
         tc.run("graph info . --version=2.0")
         assert "deprecated: None" in tc.out
+
+class TestDeprecatedIncludesMessages:
+    @pytest.mark.parametrize("import_stm,way", [
+        ["from conans.model.package_ref import PkgReference", "PkgReference()"],
+        ["from conans.model.package_ref import PkgReference", "PkgReference.loads('foo/1.0:123')"],
+        ["from conans.model.recipe_ref import RecipeReference", "RecipeReference()"],
+        ["from conans.model.recipe_ref import RecipeReference", "RecipeReference.loads('foo/1.0@user/channel')"]
+    ])
+    def test_deprecated_package_ref_include(self, import_stm, way):
```

Comment:
The deprecation messages were not being test directly, remove this when the models import is removed